jobbergate-core/jobbergate_core/auth/core.py

```diff
@@ -0,0 +1,316 @@
+"""
```

Comment:
We shouldn't have a module named "core" that is a part of a package named "jobbergate-core". It's too confusing.

Reply:
Good point, it was renamed to handler and the class inside it to JobbergateAuthHandler ([8328c22](https://github.com/omnivector-solutions/jobbergate/pull/228/commits/8328c2296dee7b226a46536345949df3752182db)).